stingray/gti.py

```diff
@@ -1042,6 +1080,58 @@ def time_intervals_from_gtis(gtis, chunk_length, fraction_step=1,
     return spectrum_start_times, spectrum_start_times + chunk_length
 
 
+def get_segment_bin_start(startbin, stopbin, nbin, fraction_step=1):
```

Comment:
Minor language nitpick. I feel like "get_" implies that you're retrieving information from some data. But I think here you're *generating* numbers. 

Maybe this would be less confusing if it were named something like `generate_segment_start_idx` or something like that?

Reply:
I use generate_ for generators, while this gives a list. 
I renamed this to calculate_segment_bin_start, how does it sound?